src/css/controls/flags/cast.less

```diff
@@ -30,6 +30,6 @@
 .jw-state-playing,
 .jw-state-paused {
     &.jw-flag-casting:not(.jw-flag-audio-player) .jw-display {
-        display: table;
+        display: flex;
```

Comment:
As long as you removed display table, is this needed? I see jw-display is already set to display: flex; in it's own block.